lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java

```diff
@@ -461,4 +464,34 @@ else if (element.isDirectory())
         }
         return;
     }
+
+    public static void copyFile(Path sourcePath, Path destPath) throws IOException {
```

Comment:
what about using Files.copy(Path, Path, CopyOption...)?
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#copy-java.nio.file.Path-java.nio.file.Path-java.nio.file.CopyOption...-
I think it is doing the same thing, isn't it?

Reply:
I was having trouble with StandardCopyOption.REPLACE_EXISTING - I can try it again.